src/pages/api/mock-backend/private/notes/index.ts

```diff
@@ -0,0 +1,59 @@
+/*
+ * SPDX-FileCopyrightText: 2022 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+import type { NextApiRequest, NextApiResponse } from 'next'
+import { HttpMethod, respondToMatchingRequest } from '../../../../../handler-utils/respond-to-matching-request'
+import type { Note } from '../../../../../api/notes/types'
+
+const handler = (req: NextApiRequest, res: NextApiResponse): void => {
+  respondToMatchingRequest<Note>(
+    HttpMethod.POST,
+    req,
+    res,
+    {
+      content: 'new note content',
+      metadata: {
+        id: 'newNoteId',
+        version: 2,
+        viewCount: 0,
+        updatedAt: '2021-04-24T09:27:51.000Z',
+        createdAt: '2021-04-24T09:27:51.000Z',
+        updateUsername: null,
+        primaryAddress: 'new-note',
+        editedBy: [],
+        title: 'New note',
+        tags: ['hedgedoc', 'demo', 'react'],
+        description: 'Many features, such wow!',
+        aliases: [
+          {
+            name: 'new-note',
+            primaryAlias: true,
+            noteId: 'newNoteId'
+          }
+        ],
+        permissions: {
+          owner: 'tilman',
+          sharedToUsers: [
+            {
+              username: 'molly',
+              canEdit: true
+            }
+          ],
+          sharedToGroups: [
+            {
+              groupName: '_LOGGED_IN',
+              canEdit: false
+            }
+          ]
+        }
+      },
+      editedByAtPosition: []
+    },
+    201
+  )
+}
```

Comment:
You mock the creation of a note that can't be opened by the frontend (if in mock mode). 
How about returning the meta data of the feature page?

Reply:
Changed